src/tests/tests.cpp

```diff
@@ -429,6 +429,7 @@ Test::Result::Result(std::string who, std::vector<Result> downstream_results)
       merge(result, true /* ignore non-matching test names */);
    }
 
+// TODO: this should move to `StdoutReporter`
```

Comment:
This method purely implements "rendering" for the stdout logging. Hence, it might make sense to move it over to `StdoutReporter`. There's a test directly accessing this though, and I wanted to keep the change "small" for now.